frontend/src/Components/ScratchIframe.tsx

```diff
@@ -1,40 +1,119 @@
 import React from 'react';
-import { Resizable, ResizableBox } from 'react-resizable';
+import {
+  Button,
+  Icon,
+  Overlay,
+  Tooltip,
+  EmptyState,
+  //@ts-ignore
+} from "@gotitinc/design-system";
+import { v4 as uuidv4 } from 'uuid';
 
 
-export default function YoutubeIframe() {
-    let [scratchUrl, setScratchUrl] = React.useState();
+export default function ScratchIframe({
+  onCloseWindow,
+} : {
+  onCloseWindow: Function,
+}) {
+  let [scratchUrl, setScratchUrl] = React.useState();
```

Comment:
NIT: 
```suggestion
  let [scratchUrl, setScratchUrl] = React.useState("");
```
for typescript to enforce typestring